models/article-models.js

```diff
@@ -0,0 +1,20 @@
+const db = require('../db/connection');
+
+exports.selectTopics = () => {
```

Comment:
Nice to see that you're splitting your models out by database... but would a query to the topics table belong with your articles models?

Reply:
I have recently noticed this and probably not I'm a little concerned I have a lot of branches at the moment so was thinking I may need to pull that first and implement this in a file of its own the next branch. I wasn't sure what to do for the best